### PR TITLE
[AIRFLOW-XXX] Highlight code blocks

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

File: docs/howto/connection/mysql.rst [hunk 1]
```diff
@@ -39,16 +39,16 @@ Extra (optional)
...
+    * ``local_infile``: controls MySQL's LOCAL capability (permitting local data loading by
       clients). See `MySQLdb docs <https://mysqlclient.readthedocs.io/user_guide.html>`_
       for details.
-    * **unix_socket**: UNIX socket used instead of the default socket.
-    * **ssl**: Dictionary of SSL parameters that control connecting using SSL. Those
-      parameters are server specific and should contain "ca", "cert", "key", "capath",
-      "cipher" parameters. See
+    * ``unix_socket``: UNIX socket used instead of the default socket.
+    * ``ssl``: Dictionary of SSL parameters that control connecting using SSL. Those
+      parameters are server specific and should contain ``ca``, ``cert``, ```key``, ``capath``,
```
Comment (feluelle) [inline docs/howto/connection/mysql.rst]:
```suggestion
      parameters are server specific and should contain ``ca``, ``cert``, ``key``, ``capath``,
```

File: docs/howto/connection/oracle.rst [hunk 2]
```diff
@@ -45,19 +45,19 @@ Extra (optional)
...
       If not specified, the environment variable ``NLS_NCHAR`` is used. If the environment
       variable ``NLS_NCHAR`` is not used, the environment variable ``NLS_LANG`` is used instead,
       and if the environment variable ``NLS_LANG`` is not set, ``ASCII`` is used.
-    * **threaded** - Whether or not Oracle should wrap accesses to connections with a mutex.
+    * ``threaded`` - Whether or not Oracle should wrap accesses to connections with a mutex.
       Default value is False.
-    * **events** - Whether or not to initialize Oracle in events mode.
-    * **mode** - one of ``sysdba``, ``sysasm``, ``sysoper``, ``sysbkp``, ``sysdgd``, ``syskmt`` or ``sysrac``
+    * ``events`` - Whether or not to initialize Oracle in events mode.
+    * ``mode** - one of ``sysdba``, ``sysasm``, ``sysoper``, ``sysbkp``, ``sysdgd``, ``syskmt`` or ``sysrac``
```
Comment (feluelle) [inline docs/howto/connection/oracle.rst]:
```suggestion
    * ``mode`` - one of ``sysdba``, ``sysasm``, ``sysoper``, ``sysbkp``, ``sysdgd``, ``syskmt`` or ``sysrac``
```

Comment (mik-laj) [reply docs/howto/connection/oracle.rst]: Fixed. Thanks.

File: docs/howto/connection/mysql.rst (hunk 1 above)
Comment (mik-laj) [reply docs/howto/connection/mysql.rst]: Fixed. Thanks.

File: docs/howto/connection/mysql.rst [hunk 3]
```diff
@@ -39,16 +39,16 @@ Extra (optional)
     Specify the extra parameters (as json dictionary) that can be used in MySQL
     connection. The following parameters are supported:
 
-    * **charset**: specify charset of the connection
-    * **cursor**: one of "sscursor", "dictcursor, "ssdictcursor" . Specifies cursor class to be
+    * ``charset``: specify charset of the connection
+    * ``cursor**: one of ``sscursor``, ``dictcursor``, ``ssdictcursor`` . Specifies cursor class to be
```
Comment (feluelle) [inline docs/howto/connection/mysql.rst]:
```suggestion
    * ``cursor``: one of ``sscursor``, ``dictcursor``, ``ssdictcursor`` . Specifies cursor class to be
```

**Review by feluelle — COMMENTED** [review]
Sorry, forgot one. (I searched for ``` and **).

File: docs/howto/connection/mysql.rst (hunk 3 above)
Comment (mik-laj) [reply docs/howto/connection/mysql.rst]: Updated. Thanks :-D

**Comment by mik-laj** [comment]
@feluelle Can you look at this agian? It gently blocks my other activities, because it affects many files.

**Review by feluelle — APPROVED** [review]

**Pull request merged**